src/_resources/one-pager-data-playbook-making-your-gtfs-publicly-available.md

```diff
@@ -1,7 +1,7 @@
 ---
 date: "2023-07-02T17:00:00-07:00"
 title: "Data Playbook: Making your GTFS publicly available"
-asset: https://drive.google.com/file/d/1G8hp5PwaR9xUdiLkq_5tXpYWqle7qwwj/view
+asset: "Cal-ITP.Data.Playbook.1-Making.Your.GTFS.Publicly.Available.pdf"
```

Comment:
Let's remove this weird `.1`

Reply:
Done in https://github.com/cal-itp/calitp.org/pull/273/commits/c40fc935edb7a3fadcb383f482c49285a00607f7